### PR TITLE
header added

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
Created header to the app

### CODE REVIEW

**Review by Tanvir04khan — APPROVED** [review]

File: src/components/categoriesBar/CategeriesBar.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import "./_categoriesBar.scss";
```
Comment (faran4engg) [inline src/components/categoriesBar/CategeriesBar.jsx]: filename typo here

**Review by faran4engg — APPROVED** [review]
one change only, approving

File: src/components/categoriesBar/CategeriesBar.jsx (hunk 1 above)
Comment (faran4engg) [reply src/components/categoriesBar/CategeriesBar.jsx]: `src/components/categoriesBar/CategeriesBar.jsx`

Comment (Tauqeer-Ahmed-99) [reply src/components/categoriesBar/CategeriesBar.jsx]: changing it

**Pull request merged**